tools/onnx/sarif/gen_sarif.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Run this script inside its folder to generate the SARIF python object model files
+# from the SARIF schema.
+# e.g. sh gen_sarif.sh
+#
+# This script requires the jschema_to_python package to be installed.
+# To install it, run:
+#   pip install jschema_to_python
+ROOT=$(pwd)/../../..
+SARIF_DIR=torch/onnx/_internal/diagnostics/infra/sarif_om
+
+# SARIF version
+SARIF_VERSION=2.1.0
+SARIF_SCHEMA_LINK=https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/schemas/sarif-schema-2.1.0.json
+
+# Download SARIF schema
+tmp_dir=$(mktemp -d)
+sarif_schema_file_path=$tmp_dir/sarif-schema-$SARIF_VERSION.json
+wget -O $sarif_schema_file_path $SARIF_SCHEMA_LINK
+
+# TODO: A private branch of jschema_to_python was used to enable
+#       the generation to dataclasses and support annotation.
+python -m jschema_to_python \
+    --schema-path $sarif_schema_file_path \
+    --module-name torch.onnx._internal.diagnostics.infra.sarif_om \
+    --output-directory $ROOT/$SARIF_DIR \
+    --root-class-name SarifLog \
+    --hints-file-path code-gen-hints.json \
+    --force \
+    --library dataclasses \
+    -vv
+
+# Generate SARIF version file
+echo "SARIF_VERSION = \""$SARIF_VERSION"\"" > $ROOT/$SARIF_DIR/version.py
+echo "SARIF_SCHEMA_LINK = \""$SARIF_SCHEMA_LINK"\"" >> $ROOT/$SARIF_DIR/version.py
+
+# Hack to have flake8 not complain about generated code.
```

Comment:
Another way is adding the generated folder at `.flake8`'s `exclude` list to skip the folder altogether

Reply:
Will need Meta review for that. I can do it in another clean up only PR.

Reply:
a cleanup pr is ok given the time constraint